app/js/listviewrowclick.js

```diff
@@ -0,0 +1,75 @@
+var BfsListViewRowClick = BfsListViewRowClick || {};
+
+BfsListViewRowClick.addClassicMutationObserver = function () {
+  var container = document;
+  var observer = new MutationObserver(BfsListViewRowClick.addRowOnClickCheckboxEventHandlerWhenListViewRowsAdded);
+
+  observer.observe(container, {
+    childList: true,
+    attributes: false,
+    characterData: false,
+    subtree: true,
+    attributeOldValue: false,
+    characterDataOldValue: false
+  });
+}
+
+BfsListViewRowClick.addRowOnClickCheckboxEventHandlerWhenListViewRowsAdded = function (mutations) {
+  // Since this code executes on each DOM addition or removal to the document,
+  // this low-level code was added to be as performant as possible.
+  var mutationsLength = mutations.length;
+  for (var mutationIndex = 0; mutationIndex < mutationsLength; ++mutationIndex) {
+    var mutation = mutations[mutationIndex];
+    if (mutation.addedNodes &&
+        mutation.addedNodes.length > 0) {
+
+      var addedNodesLength = mutation.addedNodes.length;
+
+      for (var addedNodeIndex = 0; addedNodeIndex < addedNodesLength; ++addedNodeIndex) {
+        var addedNode = mutation.addedNodes[addedNodeIndex];
+
+        if (addedNode.className &&
+            (addedNode.className.indexOf('x-grid3-row') != -1 ||
+             addedNode.className.indexOf('dataRow') != -1 ||
+             addedNode.className.indexOf('x-grid3-row-table') != -1)) {
+          BfsListViewRowClick.addClassicRowOnClickCheckBoxEventHandler();
+          return;
+        }
+      }
+    }
+  }
+}
+
+
+BfsListViewRowClick.addClassicRowOnClickCheckBoxEventHandler = function () {
+  // Setup Listview Rows
+  $('tr.dataRow').has('td:first-child input[type="checkbox"]')
+    .on('click', BfsListViewRowClick.clickCheckboxFunction);
```

Comment:
Thankfully no because the setup views use different markup than regular object listviews.